snuba/utils/streams/topics.py

```diff
@@ -41,13 +41,15 @@ class Topic(Enum):
     GENERIC_METRICS_DISTRIBUTIONS_COMMIT_LOG = (
         "snuba-generic-metrics-distributions-commit-log"
     )
+    SEARCH_ISSUES = "search-issues"
```

Comment:
Was thinking we'd call this topic `eventstream-issue-platform`, although not too fussed either way. Just want to match this up to work in eventstream.

Reply:
Yup, renamed.